msdtc/map.jinja

```diff
@@ -9,6 +9,7 @@
   key:values here that differ from whats in defaults.yml
 ##}
 {% set osrelease_map = salt['grains.filter_by']({
+    '2012ServerR2': {},
```

Comment:
@iggy I'm open to it, but it's worth considering that it hasn't been tested on older Windows server OSes (2003R2, etc), may change in upcoming releases (2016), and definitely wouldn't work/apply to Windows desktop OSes.